### DISCUSSION TITLE
Welcome to ansible-freeipa Discussions!

### DISCUSSION THREAD

**Discussion by t-woerner** [freeipa/ansible-freeipa#1011] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DIM-91** [comment]
HI! 
this is a great module, thanks for your work!
I have a question, how is it possible that through playbooks I can for example create a new account, or manage sudo rules using an account that does not have permission to do so. 
Any account that is listed in the playbook can get IPA administrator permissions?

**Reply by t-woerner** [reply]
Hi @DIM-91, thank you.
You are using root as ansible_user and are running the plugins on a server, right? Then this is expected behaviour as with root the task is automatically getting admin credentials with the server context that is used by default for the plugins. This is the same behaviour as with the command line when the server context is used.
Please do not use the root user as long as it is not really needed.
